aten/src/ATen/native/quantized/cuda/affine_quantizer.cu

```diff
@@ -56,6 +55,138 @@ void dequantize_tensor_per_tensor_affine_cuda(
       });
 }
 
+void quantize_tensor_per_channel_affine_cuda(
```

Comment:
LG!  For the slight differences between integer and floating point zero_point, did we figure out the reason for the differences?  If yes, would be great to document them in the code.

Reply:
I believe the only difference is whether you add before/after rounding which affects things due to the rule of rounding towards even numbers when the value is .5

So in practice its not a significant difference.